.changeset/proud-bugs-smile.md

```diff
@@ -0,0 +1,8 @@
+---
+'@leafygreen-ui/tokens': minor
+---
+
+[LG-2930](https://jira.mongodb.org/browse/LG-2930)
+
+- Fixes `darkModeBackgroundColors` disabled colors
```

Comment:
made the assumption that this is still a minor change because it's an aesthetic change but please feel free to correct me if this should instead be major

Reply:
I'd even say patch. It's just a bug fix, and doesn't affect API or layout